e2e/testcases/cli_test.go

```diff
@@ -833,3 +845,192 @@ func TestNomosVetNamespaceRepo(t *testing.T) {
 		})
 	}
 }
+
+func TestCLIBugreportNomosRunningCorrectly(t *testing.T) {
+	var bugReportZipName, bugReportDirName string
+	tmpDir := nomostest.TestDir(t)
+	tw := nomostesting.New(t, nomostesting.NomosCLI)
+	opts := nomostest.NewOptStruct(nomostest.TestClusterName(tw), tmpDir, tw)
+	nomostest.FreshTestEnv(t, opts)
```

Comment:
Why not using `nomostest.New` directly? I think it should also apply to CI jobs with `--share-test-env`.

Reply:
I noticed consistent presubmit failures with `nomos` command fails at creating RestMap, and the error indicates attempts to load kubeconfig from other random test directories. The root cause points to the parallelism after several tests so using this logic but skip enableParallel.

Reply:
I see. The `nomos` command loads the kubeconfig from either the `KUBECONFIG` env, or the `.kube/config` file under the current user HOME dir: https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/client/restconfig/config.go#L43.

One last thing to try is to set the env variable in the test and then unsets it in `nt.T.cleanup`.

If we can use `nomostest.New` directly, then we don't need to create a Kind cluster, and install Config Sync on the Kind cluster in the periodic jobs, which will eventually avoid extending the CI running time.

Reply:
Done by using nt and setting KUBECONFIG before running `nomos bugreport`, thanks for help debugging!